CONFIG.md

```diff
@@ -275,6 +275,55 @@ directly to
 [grunt-contrib-compass](https://github.com/gruntjs/grunt-contrib-compass)
 for this theme.
 
+### Proxying Grunt Tasks
+
+Many themes come with their own Grunt integration. In those cases you can
+configure Grunt-Drupal-Tasks for awareness of that and remove the theme
+from most handling.
+
+In order to exclude your theme from handling, specify the `proxy` property in
```

Comment:
That seems like a potential source of confusion & complexity, since there are multiple non-default options that can individually trigger the "dispatch".

`themes.themeName.build = "grunt build"` or `themes.themeName.validate = "grunt eslint"`? If nothing is set it uses the GDT functionality. The downside of being an open shell command is that we lose the ability to pass along any other flags, such as if we wanted to pass along `--verbose`.

We could re-introduce some complexity later on if we see a need for something like that by detection of an object vs. a string:

`themes.invasion.build = { cmd: "grunt", args: "build" }`